cypress/e2e/finance/funds/test-acquisition-unit-restrictions-for-funds.cy.js

```diff
@@ -0,0 +1,99 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import Ledgers from '../../../support/fragments/finance/ledgers/ledgers';
+import Users from '../../../support/fragments/users/users';
+import Funds from '../../../support/fragments/finance/funds/funds';
+import FinanceHelp from '../../../support/fragments/finance/financeHelper';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import AcquisitionUnits from '../../../support/fragments/settings/acquisitionUnits/acquisitionUnits';
+
+describe('ui-finance: Transactions', () => {
+  const defaultfund = { ...Funds.defaultUiFund };
+  const defaultFiscalYear = { ...FiscalYears.defaultUiFiscalYear };
+  const defaultLedger = { ...Ledgers.defaultUiLedger };
+  const defaultAcquisitionUnit = { ...AcquisitionUnits.defaultAcquisitionUnit };
+  let user;
+
+  before(() => {
+    cy.getAdminToken();
+    FiscalYears.createViaApi(defaultFiscalYear)
+      .then(response => {
+        defaultFiscalYear.id = response.id;
+        defaultLedger.fiscalYearOneId = defaultFiscalYear.id;
+
+        Ledgers.createViaApi(defaultLedger)
+          .then(ledgerResponse => {
+            defaultLedger.id = ledgerResponse.id;
+          });
+      });
+    cy.createTempUser([
+      permissions.uiFinanceFinanceViewGroup.gui,
+      permissions.uiFinanceCreateAllocations.gui,
+      permissions.uiFinanceCreateTransfers.gui,
+      permissions.uiFinanceExecuteFiscalYearRollover.gui,
+      permissions.uiFinanceExportFinanceRecords.gui,
+      permissions.uiFinanceManageAcquisitionUnits.gui,
+      permissions.uiFinanceManuallyReleaseEncumbrance.gui,
+      permissions.uiFinanceViewFiscalYear.gui,
+      permissions.uiFinanceViewFundAndBudget.gui,
+      permissions.uiFinanceViewGroups.gui,
+      permissions.uiFinanceViewLedger.gui,
+      permissions.uiFinanceViewEditFiscalYear.gui,
+      permissions.uiFinanceViewEditFundAndBudget.gui,
+      permissions.uiFinanceViewEditGroup.gui,
+      permissions.uiFinanceViewEditLedger.gui,
+      permissions.uiFinanceViewEditCreateFiscalYear.gui,
+      permissions.uiFinanceViewEditCreateFundAndBudget.gui,
+      permissions.uiFinanceCreateViewEditGroups.gui,
+      permissions.uiFinanceViewEditCreateLedger.gui,
+      permissions.uiFinanceViewEditDeleteFiscalYear.gui,
+      permissions.uiFinanceViewEditDeletFundBudget.gui,
+      permissions.uiFinanceViewEditDeletGroups.gui,
+      permissions.uiFinanceViewEditDeleteLedger.gui,
+      permissions.uiFinanceAssignAcquisitionUnitsToNewRecord.gui
+    ])
+      .then(userProperties => {
+        user = userProperties;
+      });
+  });
+
+  after(() => {
+    cy.loginAsAdmin({ path:SettingsMenu.acquisitionUnitsPath, waiter: AcquisitionUnits.waitLoading });
+    AcquisitionUnits.unAssignAdmin(defaultAcquisitionUnit.name);
+    AcquisitionUnits.delete(defaultAcquisitionUnit.name);
+    cy.loginAsAdmin({ path:TopMenu.fundPath, waiter: Funds.waitLoading });
```

Comment:
there is no need to login as admin one more time. you can use cy.visit and wait while the page is loaded

Reply:
Removed